src/lashup_gm_mc.erl

```diff
@@ -5,7 +5,8 @@
 %% API
 -export([
   start_link/0,
-  multicast/2
+  multicast/2,
```

Comment:
nit: the title of the commit has a typo: s/milticast/multicast

Not important, but maybe you wanna fix when you squash.